pkg/kics/resolver_sink.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-func (s *Service) resolverSink(ctx context.Context, filename, scanID string) ([]string, error) {
+func (s *Service) resolverSink(ctx context.Context, filename, scanID string, resolveReferences bool) ([]string, error) {
```

Comment:
maybe change the name o the variable to relate to openAPI, makes it easier to understand its purpose in the future